Source/FakeItEasy.Specs/ArgumentValueFormatterSpecs.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System.Diagnostics.CodeAnalysis;
+    using FluentAssertions;
+    using Xbehave;
+
+    public static class ArgumentValueFormatterSpecs
+    {
+        [Scenario]
+        public static void GenericArgumentValueFormatterDefaultPriority(
+            IArgumentValueFormatter formatter,
+            Priority priority)
+        {
+            "Given an argument value formatter that extends the generic base"
+                .x(() => formatter = new SomeArgumentValueFormatter());
+
+            "When I fetch the Priority"
+                .x(() => priority = formatter.Priority);
+
+            "Then it should be 0"

```

Comment:
I like this. I'll incorporate it. If you'd like a refinement, @thomaslevesque, keep chatting. I'll make as many changes as we need!